stores/appStore.ts

```diff
@@ -15,12 +32,52 @@ export const useAppStore = defineStore("app", {
     metadata: undefined,
     metadataFetchError: undefined,
     metadataFetchStatus: undefined,
-    currentScenario: undefined, // Represents the scenario currently being viewed
+    currentScenario: { ...emptyScenario },
   }),
   getters: {
-    globeParameter: state => state.metadata?.parameters.find(param => param.parameterType === TypeOfParameter.GlobeSelect),
+    globeParameter: (state): Parameter | undefined => state.metadata?.parameters.find(param => param.parameterType === TypeOfParameter.GlobeSelect),
+    timeSeriesData: (state): Record<string, number[]> | undefined => state.currentScenario.result.data?.time_series,
+    capacitiesData: (state): Array<ScenarioCapacity> | undefined => state.currentScenario.result.data?.capacities,
+    interventionsData: (state): Array<ScenarioIntervention> | undefined => state.currentScenario.result.data?.interventions,
   },
   actions: {
+    async loadScenarioStatus() {
+      if (!this.currentScenario.runId) {
+        return;
+      }
+
+      const {
+        data: scenarioStatusData,
+        status: scenarioStatusFetchStatus,
+        error: scenarioStatusFetchError,
+      } = await useFetch(`/api/scenarios/${this.currentScenario.runId}/status`) as {
+        data: Ref<ScenarioStatusData>
+        status: Ref<AsyncDataRequestStatus>
+        error: Ref<FetchError>
+      };
+
+      this.currentScenario.status = {
+        data: { ...scenarioStatusData.value, runId: undefined },
```

Comment:
Why set the `runId` to undefined here? Because you don't need it, because it's already on `currentScenario`? Could always just discard it - or could check it against currentScenario before setting the status.

Reply:
Yeah, this is an attempt to discard it, because it seems untidy/duplicatey to be storing multiple copies of the runId within the store - I feel we should refer to one source of truth.

The `ScenarioStatusData` interface marks runId as an optional parameter, since the interface is used to type the response from the R API.